tests/tools/soft_sort_test.py

```diff
@@ -86,6 +86,51 @@ def test_sort_batch(self, rng: jax.random.PRNGKeyArray, topk: int):
     np.testing.assert_array_equal(xs.shape, expected_shape)
     np.testing.assert_array_equal(jnp.diff(xs, axis=axis) >= 0.0, True)
 
+  def test_multiv_cdf_quantiles(self):
+    n, d = 512, 3
+    keys = jax.random.split(jax.random.PRNGKey(0), 3)
+
+    # Set central point in sampled input measure
+    z = jax.random.uniform(keys[0], (1, d))
+
+    # Sample inputs symmetrically centered on z
+    inputs = .34 * jax.random.normal(keys[0], (n, d)) + z
```

Comment:
I wanna switch from removing the `0` in `0.34`, could you please add them here and elsewhere? I think it's more legible/less prone to mistakes.